pkg/cephmgr/rgw/user.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package rgw
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+
+	ctx "golang.org/x/net/context"
+
+	"github.com/rook/rook/pkg/cephmgr/mon"
+	"github.com/rook/rook/pkg/clusterd"
+)
+
+const (
+	builtinUserKey = "admin"
+)
+
+type user struct {
+	Keys []keyInfo `json:"keys"`
+}
+
+type keyInfo struct {
+	User      string `json:"user"`
+	AccessID  string `json:"access_key"`
+	SecretKey string `json:"secret_key"`
+}
+
+func createBuiltinUser(context *clusterd.Context) error {
+	logger.Infof("creating the built-in rgw user")
+	result, err := RunAdminCommand(context,
```

Comment:
what part of this command gives the user being created full admin access to RGW?